### PR TITLE
Minority clan follow up

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/minority_clan_issues_yes_no_amb.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        "CodeType": "Normal",
+        "MatchValues": [
+          "no"
+        ],
+        "StringValue": "no",
+        "DisplayText": "No",
+        "VisibleInCoda": true
+      },
+      {
+        "NumericValue": 1,
```
Comment (as2388) [inline code_schemes/minority_clan_issues_yes_no_amb.json]:
```suggestion
        "NumericValue": 2,
```

File: code_schemes/ws_correct_dataset.json [hunk 2]
```diff
@@ -289,6 +289,17 @@
                 "s01mag15"
             ]
         },
+        {
+            "DisplayText": "minority_clan_issues",
```
Comment (as2388) [inline code_schemes/ws_correct_dataset.json]:
```suggestion
            "DisplayText": "minority clan issues",
```

File: run_scripts/1_coda_get.sh [hunk 3]
```diff
@@ -43,6 +43,8 @@ DATASETS=(
     "recently_displaced"
     "household_language"
     "women_participation"
+    "minority_clan_issues"
+
```
Comment (as2388) [inline run_scripts/1_coda_get.sh]:
```suggestion
```

File: code_schemes/minority_clan_issues_yes_no_amb.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/minority_clan_issues_yes_no_amb.json]: Nice catch, updated thankyou!

File: code_schemes/ws_correct_dataset.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/ws_correct_dataset.json]: Nice catch, updated thankyou!

File: run_scripts/1_coda_get.sh (hunk 3 above)
Comment (IsaackMwenda) [reply run_scripts/1_coda_get.sh]: updated thanks!

**Pull request merged**